ofctrl/ofAction.go

```diff
@@ -778,12 +781,38 @@ type SetICMPv4CodeAction struct {
 	Code uint8
 }
 
-func (a *SetICMPv4CodeAction) GetActionMessage() openflow13.Action {
-	field, _ := openflow13.FindFieldHeaderByName("NXM_OF_ICMP_CODE", false)
-	field.Value = &openflow13.IcmpCodeField{Code: a.Code}
-	return openflow13.NewActionSetField(*field)
+func (a *SetICMPv4CodeAction) GetActionMessage() openflow15.Action {
+	field, _ := openflow15.FindFieldHeaderByName("NXM_OF_ICMP_CODE", false)
+	field.Value = &openflow15.IcmpCodeField{Code: a.Code}
+	return openflow15.NewActionSetField(*field)
 }
 
 func (a *SetICMPv4CodeAction) GetActionType() string {
 	return ActTypeSetICMP4Code
 }
+
+type CopyFieldAction struct {
```

Comment:
I didn't find an OFAction implementation for SetFieldAction, do you miss it?

Reply:
I had implemented the SetField without the OFAction interface. Although it would work, but would implement it again as OFAction to be consistent with other actions.